app/views/content_items/unpublishing.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if @content_item.alternative_url.present? %>
       <p class="alternative">
-        Visit: <%= link_to(@content_item.alternative_url, @content_item.alternative_url, class: "govuk-link") %>
+        <%= t("unpublishing.visit") %>: <%= link_to(@content_item.alternative_url, @content_item.alternative_url, class: "govuk-link") %>
```

Comment:
Is it worth re-using the same key here as `gone.visit`?

Reply:
Yeah, I'll use a shared key for this